doc/api/worker_threads.md

```diff
@@ -100,6 +100,27 @@ if (isMainThread) {
 }
 ```
 
+## `worker.isInternalWorker`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Is `true` if this code is running inside of an internal [`Worker`][] thread (e.g a loader thread).
+
+```js
+// loader.js
+const { isInternalWorker } = require('node:worker_threads');
+
+console.log(isInternalWorker);  // Prints 'true'.
+```
+
+```bash
+node --no-warnings --experimental-loader ./loader.js --eval ""
+```
```

Comment:
````suggestion
```bash
node --experimental-loader ./loader.js main.js
```

```cjs
// loader.js
const { isInternalWorker } = require('node:worker_threads');

console.log(isInternalWorker);  // true
```
```mjs
// loader.js
import { isInternalWorker } from 'node:worker_threads';

console.log(isInternalWorker);  // true
```

```cjs
// main.js
const { isInternalWorker } = require('node:worker_threads');

console.log(isInternalWorker);  // false
```
```mjs
// main.js
import { isInternalWorker } from 'node:worker_threads';

console.log(isInternalWorker);  // false
```
````

Reply:
Is `loader.js` enough so people will realise they have to call that as a loader to work? That's why I added the bash command, because if not that's just a filename like any other

Reply:
Hmm? The bash command is present in my suggestion (I merely moved it to the top of the code samples).

Reply:
Oh, I saw that wrong 🤦🏻

Reply:
Fixed, I had to apply it manually because of the change to `isInternalThread`

Reply:
> Oh, I saw that wrong 🤦🏻 

Happens to me all the time 😅